composer.json

```diff
@@ -57,9 +57,9 @@
         "mezzio/mezzio-aurarouter": "^3.7",
         "mezzio/mezzio-fastroute": "^3.11",
         "mezzio/mezzio-laminasrouter": "^3.9",
-        "phpunit/phpunit": "^10.5.2",
+        "phpunit/phpunit": "^10.5.9",
         "psalm/plugin-phpunit": "^0.18.4",
-        "vimeo/psalm": "^5.17.0"
+        "vimeo/psalm": "^5.21.1"
```

Comment:
maybe worth checking it again, as v5.22 has been released like ... 5h ago:
https://github.com/vimeo/psalm/releases/tag/5.22.0

Reply:
Ack - Renovate will take care of that later tho - this patch gets CI green again so renovate can do its thing